Build is broken

Can be seen here:
https://github.com/gpuweb/gpuweb/actions/workflows/publish-specs.yml
First instance:
https://github.com/gpuweb/gpuweb/runs/6588654555?check_suite_focus=true#step:4:746

Logs:

```
  [INFO] Build bikeshed document "spec/index.bs" (gh)…
    $ bikeshed --die-on=warning spec "spec/index.bs" "spec/index.bs.built.html" 
      WARNING: Multiple elements have the same ID 'biblio-html'.
      Deduping, but this ID may not be stable across revisions.
       ✘  Did not generate, due to fatal errors
    Failed
  Error: Process completed with exit code 2.
  Error: w3c/spec-prod/v2/action.yml (Line: 113, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 113, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 123, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 123, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 133, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 133, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 144, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 144, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 154, Col: 21):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 154, Col: 21): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 164, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 164, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
```

--- a/spec/index.bs
+++ b/spec/index.bs
@@ -73,6 +73,13 @@ spec: canvas; urlPrefix: https://html.spec.whatwg.org/multipage/canvas.html#
         text: canvas context mode; url: concept-canvas-context-mode
         text: OffscreenCanvas context mode; url: offscreencanvas-context-mode
         text: check the usability of the image argument; url: check-the-usability-of-the-image-argument
+spec: WEBGL-1; urlPrefix: https://www.khronos.org/registry/webgl/specs/latest/1.0/#
+    type: interface
+        text: WebGLRenderingContext; url: WEBGLRENDERINGCONTEXT
+    type: attribute; for: WebGLRenderingContext
+        text: drawingBufferColorSpace; url: DOM-WebGLRenderingContext-drawingBufferColorSpace
+    type: dictionary
+        text: WebGLContextAttributes; url: WEBGLCONTEXTATTRIBUTES
 spec: WGSL; urlPrefix: https://gpuweb.github.io/gpuweb/wgsl/#
     type: dfn
         text: location; url: input-output-locations
@@ -1737,8 +1744,25 @@ conversion, with intermediate computations having at least the precision of the 
 
 ### Color Space Conversion Elision ### {#color-space-conversion-elision}
 
-Issue: Write this section.
+If the source and destination of a color space/encoding conversion are the same, then conversion
+is not necessary. In general, if any given step of the conversion is an identity function (no-op),
+implementations **should** elide it, for performance.
 
+For optimal performance, applications **should** set their color space and encoding
+options so that the number of necessary conversions is minimized throughout the process.
+For various image sources of {{GPUImageCopyExternalImage}}:
+
+- {{ImageBitmap}}:
+    - Premultiplication is controlled via {{ImageBitmapOptions/premultiplyAlpha}}.
+    - Color space is controlled via {{ImageBitmapOptions/colorSpaceConversion}}.
+- 2d canvas:
+    - [[html#premultiplied-alpha-and-the-2d-rendering-context|Always premultiplied]].
+    - Color space is controlled via the {{CanvasRenderingContext2DSettings/colorSpace}} context creation attribute.
+- WebGL canvas:
+    - Premultiplication is controlled via the `premultipliedAlpha` option in {{WebGLContextAttributes}}.
+    - Color space is controlled via the {{WebGLRenderingContext}}'s {{WebGLRenderingContext/drawingBufferColorSpace}} state.
+
+Note: Check browser implementation support for these features before relying on them.
 
 # Initialization # {#initialization}
 

